tftp.go

```diff
@@ -0,0 +1,149 @@
+package ipxe
```

Comment:
I just noticed that this isn't in its own `tftp` package. small packages is good in go right? why not put this in its own package and then you don't need to do things like add the func to type/function names. Thats pseudo namespacing that the compiler can do better.

Reply:
I don't see any significant value in this, but also don't see any reason not to. will update.